django-cloudlaunch/cloudlaunch/backend_plugins/cloudman2/rancher2_aws_iam_trust_policy.json

```diff
@@ -4,9 +4,9 @@
         {
             "Effect": "Allow",
             "Principal": {
-                "Service": "ec2.amazonaws.com"
+                "Service": "ec2.amazonaws.com.cn"
```

Comment:
But this one is still wrong by having `.cn` added?

Reply:
Fixed at last I think! Thanks.